command/acl_bootstrap_test.go

```diff
@@ -76,3 +78,67 @@ func TestACLBootstrapCommand_NonACLServer(t *testing.T) {
 	out := ui.OutputWriter.String()
 	assert.NotContains(out, "Secret ID")
 }
+
+// Attempting to bootstrap the server with an operator provided token should
+// return the same token in the result.
+func TestACLBootstrapCommand_WithOperatorBootstrapToken(t *testing.T) {
+	ci.Parallel(t)
+	assert := assert.New(t)
+
+	// create a acl-enabled server without bootstrapping the token
+	config := func(c *agent.Config) {
+		c.ACL.Enabled = true
+		c.ACL.PolicyTTL = 0
+	}
+
+	// create a valid token
+	mockToken := mock.ACLToken()
+
+	srv, _, url := testServer(t, true, config)
+	defer srv.Shutdown()
+
+	assert.Nil(srv.RootToken)
```

Comment:
If this assertion fails (which it never should), then the whole rest of the test is invalid. This is a good use case for using `require` instead of `assert`:

```suggestion
	require.Nil(t, srv.RootToken)
```

(Applies to the test below as well.)